frontend/src/charts/SimpleHorizontalBarChart.tsx

```diff
@@ -204,6 +205,7 @@ export function SimpleHorizontalBarChart(props: SimpleHorizontalBarChartProps) {
           BREAKDOWN_VAR_DISPLAY_NAMES[props.breakdownVar],
           props.metric.metricId,
           props.metric.shortVegaLabel,
+          props.metric.type === "pct_share" ? "%" : "",
```

Comment:
Would this helper work here? https://github.com/SatcherInstitute/health-equity-tracker/blob/2c02770c7f86613830ef7569e607b82413f9c734/frontend/src/data/config/MetricConfig.ts#L93

Reply:
It would be ideal to use this helper yeah..I gave it a brief try but I wasn't able to figure out how to make it callable within the signal in SimpleHorizontalBarChart; if you know of an easy way to do this, lmk, otherwise will just submit as-is